docs/src/standard_results/field_outline.md

```diff
@@ -0,0 +1,88 @@
+# Field outline
+
+This exports the modelled field outline from within RMS.
+This field outline is a polygon representing the outline of the hydrocarbon-filled
+reservoir under initial conditions.
+
+The field outline is typically calculated as the intersection between the modelled
+top reservoir surface (in depth) and the fluid contact surface. Some assets may
+choose to further refine the outline by limiting it to areas with `net` reservoir
+properties. It is up to each asset to decide which definition best suits their field.
+
```

Comment:
Ref comments further down; perhaps add something like: The intention is to export field outlines calculated for every realization, however the calculation of the outline is not scope for the export function.

...you can probably phrase this better than me, but what I think should come across is that the user should probably not just dump a static polygon into this folder and then start exporting (the same) polygon for every realization.